serveradmin/servershell/views.py

```diff
@@ -197,8 +198,21 @@ def get_results(request):
 
 @login_required
 def inspect(request):
-    server = Query({'object_id': request.GET['object_id']}, None).get()
-    return _edit(request, server, template='inspect')
+    if request.method != 'GET':
```

Comment:
There are the http decorators, which you can use for that: https://docs.djangoproject.com/en/2.2/topics/http/decorators/

Reply:
Nice - didn't not stumble across them yet.